docs/GettingStartedDocs/buildandsign.md

```diff
@@ -97,40 +97,9 @@ NumHeapPages=1024
 NumStackPages=1024
 NumTCS=1
 ```
-Additionally, a developer can specify additional Key Sharing and Separation (KSS) identity properties
```

Comment:
Is this GCC specific? trying to understand the deletion.

Reply:
Good catch, this shouldn't have been changed.